apps/desktop/desktop/src/main/java/bisq/desktop/main/content/mu_sig/offerbook/MuSigOfferbookController.java

```diff
@@ -226,8 +231,24 @@ public void clear() {
         });
         model.getSortedMarketItems().setComparator(model.getSelectedMarketSortType().get().getComparator());
 
+        Direction persistedOfferlistFilter = settingsService.getCookie().asString(CookieKey.MU_SIG_OFFERLIST_FILTER)
+                .map(name -> ProtobufUtils.enumFromProto(Direction.class, name, null))
```

Comment:
I think to use a specific enum covering the 2 cases is better than to use null for the 3rd case.